src/Sarif.Viewer.VisualStudio/Tags/SarifLocationTagHelpers.cs

```diff
@@ -1,25 +1,30 @@
-﻿// Copyright (c) Microsoft. All rights reserved. 
-// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Text;
 
 namespace Microsoft.Sarif.Viewer.Tags
 {
-    using Microsoft.VisualStudio.ComponentModelHost;
-    using Microsoft.VisualStudio.Shell;
-
     internal static class SarifLocationTagHelpers
     {
         /// <summary>
         /// Calls into the tagger service and asks it to refresh the tags being displayed in Visual Studio.
         /// </summary>
-        public static void RefreshAllTags()
+        /// <param name="textBuffer">
+        /// The text buffer whose tags are to be refeshed, or null if the tags for all text buffers
+        /// are to be refreshed.
+        /// </param>
+        public static void RefreshAllTags(ITextBuffer textBuffer = null)
```

Comment:
In a "perfect" world, we would send the span for which the tags need to be refreshed.. but that may be WAY overkill (unless the file is like a large number of lines long)

Reply:
Per discussion, merging without change.

---
In reply to: [516216986](https://github.com/microsoft/sarif-visualstudio-extension/pull/245#discussion_r516216986) [](ancestors = 516216986)